From: Jeremy Bicha <jbicha@debian.org>
Date: Tue, 15 Feb 2022 09:46:23 -0500
Subject: Support GNOME 42 dark theme preference

---
 meson.build          | 2 +-
 src/application.vala | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 032f16d..85c2a83 100644
--- a/meson.build
+++ b/meson.build
@@ -19,7 +19,7 @@ gweather = dependency('gweather-3.0', version: '>=3.32.0')
 gnomedesktop = dependency('gnome-desktop-3.0', version: '>=3.8')
 geocodeglib = dependency('geocode-glib-1.0', version: '>=1.0')
 libgeoclue = dependency('libgeoclue-2.0', version: '>=2.4')
-libhandy = dependency ('libhandy-1', version: '>= 1.0.0', required: false)
+libhandy = dependency ('libhandy-1', version: '>= 1.5.90', required: false)
 
 libhandy_subproj = libhandy
 if not libhandy.found()
diff --git a/src/application.vala b/src/application.vala
index 2e67046..bc9bc60 100644
--- a/src/application.vala
+++ b/src/application.vala
@@ -116,6 +116,8 @@ public class Application : Gtk.Application {
         base.startup ();
 
         Hdy.init ();
+        Hdy.StyleManager.get_default().set_color_scheme(
+            Hdy.ColorScheme.PREFER_LIGHT);
 
         Utils.load_css ("gnome-clocks");
 
